app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -1,60 +1,68 @@
-    <% end %>
-  <% end %>
-  <div class="links">
-    <%= render partial: "shared/authentication/providers_registration_form" %>
 
-    <div id="sign-in-password-form" class="mt-1 crayons-card p-7 align-left mx-auto" style="max-width:490px;">
-      <p class="pb-4 fw-bold">If you have a password...</p>
-      <%= form_for(User.new, as: :user, url: session_path(:user)) do |f| %>
-        <div class="crayons-field">
-          <%= f.label :email, class: "crayons-field__label" %>
-          <%= f.email_field :email, autocomplete: "email", required: true, class: "crayons-textfield" %>
-        </div>
+    <div class="registration__actions">
+      <%= render partial: "shared/authentication/providers_registration_form" %>
 
-        <div class="field">
-          <%= f.label :password, class: "crayons-field__label" %>
-          <%= f.password_field :password, autocomplete: "current-password", required: true, class: "crayons-textfield" %>
+      <div class="registration__actions-email" id="sign-in-password-form">
+        <div class="registration__hr">
+          <span class="registration__hr-label">
+            Or continue with your email address
+          </span>
         </div>
+        <%= form_for(User.new, as: :user, url: session_path(:user)) do |f| %>
+          <div class="crayons-field mb-2">
+            <%= f.label :email, class: "crayons-field__label" %>
+            <%= f.email_field :email, autocomplete: "email", class: "crayons-textfield" %>
+          </div>
 
-        <div class="field">
-          <%= f.check_box :remember_me %>
-          <%= f.label :remember_me %>
-        </div>
+          <div class="crayons-field mb-2">
+            <%= f.label :password, class: "crayons-field__label" %>
+            <%= f.password_field :password, autocomplete: "current-password", class: "crayons-textfield" %>
+          </div>
 
-        <div class="actions pt-3">
-          <%= f.submit "Log in", class: "crayons-btn" %>
-        </div>
-      <% end %>
-      <% if any_selfserve_auth? %>
-        <p class="pt-6 fs-s"><em>Sign in by social auth if you don't have a password set.</em></p>
-      <% end %>
-        <p class="pt-6 fs-s"><em><a href="<%= new_password_path(:user) %>">Forgot password?</a></em></p>
+          <div class="field">
+            <%= f.check_box :remember_me, class: "crayons-checkbox" %>
+            <%= f.label :remember_me, class: "fs-s" %>
+          </div>
+
+          <div class="actions pt-3">
+            <%= f.submit "Continue", class: "crayons-btn crayons-btn--l w-100" %>
+          </div>
+        <% end %>
+        <p class="pt-6 fs-s align-center">
+          <a href="<%= new_password_path(:user) %>">
+            I forgot my password
+          </a>
+        </p>
+      </div>
     </div>
   </div>
-  <p>Open Source 😇</p>
-  <p>Free Forever ❤️</p>
-  <div style="height:300px"></div>
-</div>
+  <footer class="registration__footer">
+    <p>
+      Open Source 😇 · Free Forever ❤️
```

Comment:
Having emoji in one place and regular character for heart in another makes me anxious :D 

Also, we're leaning towards using open source library for emojis: [twemoji](https://twemoji.twitter.com/) - we've been using these for icon in the main navigation in left sidebar. Maybe we could keep using them here as well - that way we don't have to rely on [particular platform](https://emojipedia.org/smiling-face-with-halo/) for showing the emoji.

Reply:
Oh awesome, thanks! I've never used this before, so I'll see how far along I get and reference the main nav for help.